ui/raidboss/popup-text.js

```diff
@@ -455,7 +553,14 @@ class PopupText {
       // on (speech=true, text=true, sound=true) but this will
       // not cause tts to play over top of sounds or noises.
       if (ttsText && playSpeech) {
-        ttsText = ttsText.replace(/[#!]/, '');
+        // Heuristics for auto tts.
+        // * Remove a bunch of chars.
+        ttsText = ttsText.replace(/[#!\/]/, '');
```

Comment:
It's a little weird to have these changes in here too.  Can you rebase on top of the latest master?